stackrox/stackrox
# ROX-13240: fix scanner-slim updates when WebSockets are used

## Description

Scanner-slim was unable to update its repository-to-cpe.json file in WebSocket environments due to a request construction error. This PR fixes this error.

## Checklist

## Testing Performed

CI as well as manual testing.

Manual testing:

Start up two OpenShift clusters, and deploy the Operator version associated with this PR to each. Deploy Central to one of the clusters and SecuredCluster to the other, all via the Operator. This will create a Scanner Slim in the SecuredCluster. Wait some time, and you'll see Scanner is able to reach out to Central (via Sensor) without error.

```
{"Event":"starting slim update","Level":"info","Location":"slim_updater.go:114","Time":"2022-11-14 20:02:16.629097"}
{"Event":"already up-to-date, nothing to do","Level":"info","Location":"slim_updater.go:127","Time":"2022-11-14 20:02:16.646015"}
{"Event":"starting slim update","Level":"info","Location":"slim_updater.go:114","Time":"2022-11-14 20:07:16.629638"}
{"Event":"already up-to-date, nothing to do","Level":"info","Location":"slim_updater.go:127","Time":"2022-11-14 20:07:16.646152"}
```
